### PR TITLE
Resolve: 282 add and change documentation

### LINKED ISSUE #282: Add and change documentation
We currently have no `contributing.md` and our `readme` is looking quite bland. Time to fix both.

### PR DESCRIPTION
**Pull request by Mitchell3514**
- Changed the `contributing.md` in .github.
- Changed the readme file to include more information and styled it differently.

Resolves #282

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+  - What is the current situation?
+  - Why should this be changed?
+  - What changes are necessary?
+- Add any relevant tags.
+- If you are planning to resolve the issue yourself, at least partly, then assign yourself to the issue.
+
+### Commenting
+- All comments should be written in English.
+- Be sure to follow our [Code of Conduct].
+- Keep any discussion on-topic, or move it to another issue/platform.
```
Comment (Rexogamer) [inline .github/CONTRIBUTING.md:28]: Should we add a link to the dev server?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,2 +1,59 @@
...
+
+### Creation
+- Keep the title short, descriptive and [imperative].
+- Write a short description of your changes.  
+If your PR solves any issue(s), make sure to add `resolves #<issue nr>` at the end.
+- Add any relevant tags.
+
+### Review
+These steps have been written to go from quick and easy to more rigorous.  
+If at any stage you find an issue, you can choose to stop at that stage until a fix has been pushed. Do make sure to go through the entire process of the stage you found an issue at.
```
Comment (Rexogamer) [inline .github/CONTRIBUTING.md:40]: Maybe add something like "If you need any help testing your changes, let us know and a maintainer/contributor will help"?

**Review by Rexogamer — APPROVED** [review]
LGTM, two non-blocking comments

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Mitchell3514) [reply .github/CONTRIBUTING.md:28]: Yeah that wouldn't be bad.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (Mitchell3514) [reply .github/CONTRIBUTING.md:40]: Only org members/collaborators can review anyway. That section is really only for those people.
I did think of adding something like "when you are ready, request review", but they wouldn't know who to ask. You just have to add the 'finished' tag when you want review.

Comment (Mitchell3514) [reply .github/CONTRIBUTING.md:40]: Alright I just checked and it seems like you can request a team to review, I'll put us in a team so they can request all of us at once.

Comment (Rexogamer) [reply .github/CONTRIBUTING.md:40]: I could be wrong, but I don't think external contributors can add tags?

Comment (Mitchell3514) [reply .github/CONTRIBUTING.md:40]: Oh that could be true also

**Review by SunburntRock89 — APPROVED** [review]

**Pull request merged**